apps/site/assets/css/_stop-page.scss

```diff
@@ -215,6 +215,18 @@
   }
 }
 
+.m-stop-page__departures--small-screen {
+  @include media-breakpoint-up(md) {
+    display: none;
+  }
+}
+
+.m-stop-page__departures--regular-screen {
+  @include media-breakpoint-down(sm) {
+    display: none;
+  }
+}
+
```

Comment:
**suggestion:** I believe the version of bootstrap we're on includes utility classes for hiding/unhiding elements based on breakpoint, in the format `"hidden-**-up"` or `"hidden-**-down"` where ** is the breakpoint name. Give them a try and you might not need this CSS!